recipes/s3fs-fuse/environment.yml

```diff
@@ -0,0 +1,13 @@
+channels:
```

Comment:
This file should be removed I think.

Reply:
that was unintentional.. thanks @bgruening!!